exercises/L-functions-nested/exercise2.js

```diff
@@ -0,0 +1,23 @@
+var mentor1 = "Daniel";
+var mentor2 = "Irina";
+var mentor3 = "Mimi";
+var mentor4 = "Rob";
+var mentor5 = "Yohannes";
+function upperCase(name) {
+  return name.toUpperCase();
+}
+function createGreeting(name) {
+  var upName = upperCase(name);
+  var message = "HELLO " + upName;
```

Comment:
I also think, your function 'createGreeting' can have two parameters, and the reason for that is what if we wanted to change the word 'HELLO' to something else later in future. And to do that you can write it like this : 
```
function createGreeting(greeting, name){
    var upName = upperCase(name);
    var message = greeting.toUpperCase( ) + upName;
    return message;
};
```